index.html

```diff
@@ -319,6 +318,27 @@ <h2 class="about-title">About<br />Myself<br /></h2>
         <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
       </div>
     </section>
+    <section class="contact" id="contact">
+      <div class="inner-div">
+        <h2>Contact me</h2>
+        <p id="inner-div">If you have an application you are interested in developing, a feature that you need
+          built or a project that needs coding. I'd love to help with it </p>
+      </div>
+      <form action="https://formspree.io/f/mvoljlod" id="form" class="form" method="post">
```

Comment:
There is some white space at the bottom of the page. Consider fixing it.
![image](https://user-images.githubusercontent.com/33268927/178980658-50568b8d-c679-4a70-85e5-d6c319c73bf8.png)